### PR TITLE
Use standard six package instead of a local modified version

### PR DESCRIPTION
**Pull request by stuertz**
Refactored to use the widespread six package, which is also used by the dependency python-utils.

### CODE REVIEW

File: progressbar/utils.py [hunk 1]
```diff
@@ -384,21 +384,22 @@ def format_time(time, precision=datetime.timedelta(seconds=1)):
     '''
     precision_seconds = precision.total_seconds()
 
-    if isinstance(time, six.basestring + six.numeric_types):
+    if isinstance(tval, six.string_types + six.integer_types + (float, )):
         try:
-            time = datetime.timedelta(seconds=six.long_int(time))
+            castfunc = long if six.PY2 else int  # NOQA
```
Comment (WoLpH) [inline progressbar/utils.py]: Might be easier to simply use: `castfunc = six.integer_types[-1]`

Comment (stuertz) [reply progressbar/utils.py]: Not sure if six guarantees the order, have to reread that.

Comment (WoLpH) [reply progressbar/utils.py]: Not sure if it strictly does, but the tuple hasn't changed in 7 years so it seems pretty stable :)
https://github.com/benjaminp/six/blame/master/six.py#L50

Comment (WoLpH) [reply progressbar/utils.py]: Alternatively... `type(2**64)` ;)

**Comment by WoLpH** [comment]
I've reported the current issue upstream: https://github.com/pytest-dev/pytest-runner/issues/36

For the time being it might be easier to simply ignore these errors for now. I think adding `W191 ptr.py` to `pytest.ini` might do the trick: https://github.com/WoLpH/python-progressbar/blob/develop/pytest.ini#L16-L19

**Comment by WoLpH** [comment]
I've merged the changes and pushed them to dev, thanks for all the help :)

**Comment by stuertz** [comment]
Thanks for accepting my change.

**Pull request merged**